src/components/Person/Person.jsx

```diff
@@ -1 +1,38 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+function Person({ personalInfo }) {
+  const {
+    name,
+    age,
+    partnerName,
+    isMarried,
+    sex,
+  } = personalInfo;
+
+  const personPartnerElement = (
+    <>
+      {sex === 'm' ? (
+        `My wife's name is ${partnerName}`
+      ) : (
+        `My husband's name is ${partnerName}`
+      )}
+    </>
+  );
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is {name}
+      </h2>
+      <p className="Person__age">
+        {age && `I am ${age}`}
```

Comment:
You have [I am](https://imgur.com/a/T6oiZGP) although age is empty 

```suggestion
        { age && (
          <p className="Person__age">
            {`I am ${age}`}
          </p>
        )}
```